sql_2g/derived_tables/licenses.sql

```diff
@@ -0,0 +1,64 @@
+DROP TABLE IF EXISTS folio_reporting.licenses;

```

Comment:
Yes, I think `licenses_ext.sql` is a good idea, and in general using prefixes like `licenses_`.

Reply:
Thanks @nassibnassar ! Just thinking of the derived tables that joins tables from two different modules 
i.e. `purchase_order_transactions.sql` would become `orders_purchase_order_finance_transactions.sql` ?

Reply:
I think we can't do this in a purely mechanical way, or the names will get too long.  I think using the "starting" schema/table name as a prefix would help with organizing them, and then adding either "ext" for general extension of the data or some other name maybe indicating the most salient join would be helpful to convey what the derived table is about.

Reply:
So on the whole I would think about how to concisely describe the derived table, and use a prefix that helps organize them so that alphabetically related tables are listed together.